src/AnsiUtils.ps1

```diff
@@ -58,12 +58,12 @@ function Get-VirtualTerminalSequence ($color, [int]$offset = 0) {
 
     if ($color -is [String]) {
         try {
-            if ($null -ne ($color -as [System.ConsoleColor])) {
-                $color = [System.ConsoleColor]$color
-            }
-            elseif ($ColorTranslatorType) {
+            if ($ColorTranslatorType) {
                 $color = $ColorTranslatorType::FromHtml($color)
             }
+            elseif ($null -ne ($color -as [System.ConsoleColor])) {
+                $color = [System.ConsoleColor]$color
```

Comment:
This is the anti-pattern of converting twice - you should either use `-as` or the cast, but not both.

Reply:
OK, caught me being lazy but at least we fixed the bug where `[System.ConsoleColor]::Black` (value of 0) was not registering as a ConsoleColor.  :-)